app/views/articles/tags/_sidebar_additional.html.erb

```diff
@@ -2,7 +2,7 @@
   <div id="sidebar-wrapper-right" class="sidebar-wrapper sidebar-wrapper-right">
     <div class="sidebar-bg" id="sidebar-bg-right"></div>
     <div class="side-bar sidebar-additional showing" id="sidebar-additional">
-      <% active_threads = Article.active_threads([@tag, "discuss"], Timeframe.datetime(params[:timeframe])) %>
+      <% active_threads = Articles::ActiveThreadsQuery.call(relation: Article.published, options: { tags: [@tag, "discuss"], time_ago: Timeframe.datetime(params[:timeframe]) }) %>
```

Comment:
You can leave off the `relation` here, as it's the same as the default argument:

```suggestion
      <% active_threads = Articles::ActiveThreadsQuery.call(options: { tags: [@tag, "discuss"], time_ago: Timeframe.datetime(params[:timeframe]) }) %>
```

Reply:
Good call! ✅